### DISCUSSION TITLE
gojs 无法通过qiankun加载成功，这个有人有试过吗？

### DISCUSSION THREAD

**Discussion by leroy-chen** [umijs/qiankun#2359] [Q&A]
无法在qiankun中加载使用gojs写的页面，报错如下：
![image](https://user-images.githubusercontent.com/8088746/208074263-a8550bc9-61cc-46c2-8e21-a2304e2dcd57.png)
页面有多个TAB时，会导致其他内容也加载不出来，修改go-module.js中的内容（注释2240行，新增2241行）后，其他内容可以加载出来，但是gojs使用不了。
![image](https://user-images.githubusercontent.com/8088746/208074600-ba2c77a2-a886-4876-a0fe-d8a0ae18024b.png)

**Comment by yexiuqing** [comment]
我也遇到到了同样的问题，有解法吗
